x-pack/solutions/observability/plugins/infra/public/pages/metrics/hosts/hooks/use_host_count.ts

```diff
@@ -48,9 +57,12 @@ export const useHostCount = () => {
         total: data.count ?? 0,
         with_query: !!searchCriteria.query.query,
         with_filters: searchCriteria.filters.length > 0 || searchCriteria.panelFilters.length > 0,
+        schema_selected: searchCriteria?.preferredSchema || 'ecs',
+        schemas_available: schemas,
+        schema_error: timeRangeMetadataStatus === FETCH_STATUS.FAILURE,
```

Comment:
Do we want to capture API errors, or if, for instance, the current selected schema doesn't have data?

Reply:
As agreed we keep it as it is, true or false for api errors